spec/builders/v2/report_builder_spec.rb

```diff
@@ -247,6 +259,19 @@
         expect(metrics[:avg_resolution_time]).to be 0
         expect(metrics[:resolutions_count]).to be 0
       end
+
+      it 'returns argument error for incorrect group by' do
+        params = {
+          type: :label,
+          id: label_2.id,
+          since: (Time.zone.today - 3.days).to_time.to_i.to_s,
+          until: Time.zone.today.to_time.to_i.to_s,
+          group_by: 'test'.to_s
+        }
+
+        builder = V2::ReportBuilder.new(account, params)
+        expect { builder.summary }.to raise_error(ArgumentError)
```

Comment:
Can we have the tests against the correct group, with a metrics check?

Reply:
Done